internals/scripts/clean.js

```diff
@@ -38,6 +38,7 @@ cp('internals/templates/homePage/messages.js', 'app/containers/HomePage/messages
 mkdir('-p', 'app/translations');
 cp('internals/templates/translations/en.json',
   'app/translations/en.json');
+rm('app/translations/de.json');
```

Comment:
Let's put this above the `mkdir('-p', 'app/translations');` and delete _all_ translations with `rm('-rf', 'app/translations/*')`